client/lib/gsuite/index.js

```diff
@@ -13,7 +13,19 @@ import userFactory from 'lib/user';
 /**
  * Constants
  */
-const GOOGLE_APPS_LINK_PREFIX = 'https://mail.google.com/a/';
+const GSUITE_LINK_PREFIX = 'https://mail.google.com/a/';
+
+/**
+ * Applies a precision to the cost
```

Comment:
We could clarify what a precision is here, maybe by adding a simple example?

Reply:
I'm going to delete this in another PR since it is not used anywhere